distribution/src/resources/drill-override-example.conf

```diff
@@ -214,7 +214,7 @@ drill.exec: {
 }
 
 # Below SSL parameters need to be set for custom transport layer settings.
-javax.net.ssl {
+ssl{
   keyStore: "/keystore.file",
```

Comment:
This is an existing property. Generally, a leading slash means an absolute path. So, do we expect the keystore to be in the Linux root directory? This would be very unusual. Or, do we mean "keystore.file" relative to some implied root directory? What is that root?

If we do not have a root, should we look relative to the $DRILL_CONF directory (normally /usr/drill or $DRILL_HOME/conf or given by the --site option.)